src/auth/auth-cross-domain-iframe.ts

```diff
@@ -0,0 +1,153 @@
+//#region imports
+
+import {
+  BBAuthDomUtility
+} from '../shared/dom-utility';
+
+import {
+  BBAuthNavigator
+} from '../shared/navigator';
+
+import {
+  BBAuthGetTokenArgs
+} from './auth-get-token-args';
+
+import {
+  BBAuthTokenError
+} from './auth-token-error';
+
+import {
+  BBAuthTokenErrorCode
+} from './auth-token-error-code';
+
+import {
+  BBAuthTokenResponse
+} from './auth-token-response';
+
+//#endregion
+
+const URL = 'https://s21aidntoken00blkbapp01.nxt.blackbaud.com/Iframes/CrossDomainAuthFrame.html'; // URL to get IFrame
```

Comment:
Is there not a more "friendly" URL we can bake into the source code? That random string in the domain makes me think this URL could change in the future?